cms/static/js/programs/models/api_config_model.js

```diff
@@ -0,0 +1,17 @@
+define([
+        'backbone'
+    ],
+    function( Backbone ) {
+        'use strict';
+
+        return Backbone.Model.extend({
+            defaults: {
+                username: '',
+                lmsBaseUrl: 'http://127.0.0.1:8001/',
+                programsApiUrl: 'http://127.0.0.1:8004/',
+                authUrl: 'http://127.0.0.1:8001/programs/id_token/',
```

Comment:
Since the app is now hosted exclusively on Studio, you could default this to just `/programs/id_token/`.